src/common/dao/group/usergroup.go

```diff
@@ -172,3 +165,12 @@ func OnBoardUserGroup(g *models.UserGroup) error {
 	}
 	return onBoardCommonUserGroup(g, "GroupName", "GroupType")
 }
+
+// GetModelsFromName ...
+func GetModelsFromName(groupNames []string, groupType int) []models.UserGroup {
```

Comment:
If this is a exported func, wouldn't it be moved to models pkg? as it has nothing to do with DB operation?